extensions/system-monitor/src/system-monitor.tsx

```diff
@@ -0,0 +1,27 @@
+import { List } from "@raycast/api";
+import { cpuUsage, freemem, freememPercentage } from "os-utils";
+import { useState } from "react";
+
+export default function SystemMonitor() {
+  const [cpu, setcpu] = useState(0);
+  const [freeMemPercentage, setFreeMemPercentage] = useState(0);
+  const [freeMem, setFreeMem] = useState(0);
+
+  cpuUsage((v) => {
+    setcpu(Math.round(v * 100));
+    setFreeMemPercentage(Math.round(freememPercentage() * 100));
+    setFreeMem(Math.round(freemem() / 1024));
+  });
+
+  const render = () => {
+    if (!cpu) return <List.Item title="Calculating..." />;
```

Comment:
instead of a item, you should use the `isLoading` prop of the List component, it'll be much nicer

Reply:
Done.